src/Entity/Reflection/AnnotationParser.php

```diff
@@ -178,13 +182,34 @@ protected function parseAnnotationValue($name, array $types, $access, $params)
 			preg_match_all('#\{([^}]+)\}#i', $params, $matches, PREG_SET_ORDER);
```

Comment:
Tohle je ještě špatně, tj. `{default "}"}` nebude fungovat, ale nemám čas to teď opravovat.